### PR TITLE
Create LICENSE

### LINKED ISSUE #118: Add a License
Companies a lot of times through security scans will not allow a library to be used if it does not provide a license, specifically a MIT license. This should definitely be the case here as this is a pretty popular library used by some frameworks.

### PR DESCRIPTION
**Pull request by chris-washington**
Resolves #118

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
+
+Copyright (c) 2018 webidl2js <npm@smayr.name>
```
Comment (domenic) [inline LICENSE]: This isn't a person's name

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+ The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
```
Comment (domenic) [inline LICENSE]: Extra space

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+Copyright (c) 2018 webidl2js <npm@smayr.name>
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+ The above copyright notice and this permission notice shall be included in
```
Comment (domenic) [inline LICENSE]: Extra space; missing newline

File: LICENSE (hunk 1 above)
Comment (TimothyGu) [reply LICENSE]: Done.

**Comment by TimothyGu** [comment]
@domenic I've fixed all your comments. Will merge soon.

**Pull request merged**